changelog/unreleased/4404

```diff
@@ -1,7 +1,9 @@
-Enhancement: Improvements in remove dialog alert
+Enhancement: Improvements in remove dialog
 
-A custom dialog alert has been added when the file that is going to be deleted has thumbnail.
+The dialog has been modified when the file that is going to be deleted has thumbnail.
```

Comment:
We don't know which dialog this is referring to, you can't write "The dialog...". Also, now we use this dialog no matter whether the file has thumbnail or not. Please, take a look at this carefully